meta-refkit/recipes-image/images/refkit-image-computervision.bb

```diff
@@ -6,7 +6,13 @@ REFKIT_IMAGE_COMPUTERVISION_EXTRA_INSTALL ?= "${REFKIT_IMAGE_INSTALL_COMMON}"
 REFKIT_IMAGE_EXTRA_FEATURES += "${REFKIT_IMAGE_COMPUTERVISION_EXTRA_FEATURES}"
 REFKIT_IMAGE_EXTRA_INSTALL += "${REFKIT_IMAGE_COMPUTERVISION_EXTRA_INSTALL}"
 
-REFKIT_IMAGE_COMPUTERVISION_EXTRA_INSTALL_append = " opencv gstreamer1.0-vaapi"
+FEATURE_PACKAGES_computervision += "packagegroup-computervision"
```

Comment:
The feature definition you mean? I can do that too if you think that's better. It's a tradeoff between encapsulating the computer vision stuff and making it easier to use several features in one image.